.github/workflows/image-scan.yml

```diff
@@ -44,7 +44,7 @@ jobs:
     steps:
     - name: Call Lacework scan
       id: run-scan
-      uses: portworx/workflow-image-scan/.github/actions/action-lacework-scan
+      uses: portworx/workflow-image-scan/.github/actions/action-lacework-scan@v1
```

Comment:
Maybe this could just reference it locally and not care about tags?
Something like 
```yaml
uses: ./github/actions/action-lacework-scan
```

Reply:
Clarified offline, the workflow doesn't have anything checked out so this doesn't work - not worth adding the checkout just for this probably.